app/assets/javascripts/pages/data_portal/DataPortalCountryPage.js

```diff
@@ -39,7 +39,8 @@
 
   // This object is used to detect the category of the indicators without having to repeat
   // the exact name
-  // NOTE: this object is duplicated in ChartWidgetView; make sure to update both of them
+  // NOTE: this object is duplicated in ChartWidgetView and ApplyFiltersView; make sure to
+  // update both of them
```

Comment:
Wouldn't be better have it as a helper? This way it doesn't need to be changed twice. In case they have different values we can separate their categories with a namespace or similar.